### DISCUSSION TITLE
Can I use time series data with multiple replicates per time point

### DISCUSSION THREAD

**Discussion by revolvefire** [aertslab/scenicplus#71] [Q&A]
I have time series data with multiple replicates per time point

I understand that multiple samples can be used but can I inter-compare between different samples from different time points?

Also, would it be okay to use multiomics data and seperate scRNA+scATAC data with label transfer together.

Thanks for the wonderful tool!

HJ

**Comment by SeppeDeWinter** [comment]
Hi @revolvefire 

I moved your issue to the discussions tab because it's a question and not a bug in the code.

```
(...) can I inter-compare between different samples from different time points?
```

What exactly do you mean by intercompare? What do you want to compare?

One thing you can compare is the AUC values (i.e. eRegulon enrichment scores) across different samples of different time points.
Also, take note that a _single_ eRegulon (for each TF) is generated across the dataset, with this I mean that one eRegulon might contain target genes/regions which are expressed/accessible in different samples (i.e. it's not the case that you get multiple eRegulons, one for each sample).

```
Also, would it be okay to use multiomics data and seperate scRNA+scATAC data with label transfer together.
```

This is possible to do, however it will require some custom code.  One easy way is to first create a scenicplus object with the non-multiome data. This will generate a pseudo-multiome dataset. The you can concat the gene expression matrix (`scplus_obj.to_df('EXP')`) and region accessbility matrix ((`scplus_obj.to_df('ACC')`) of the pseudo-multiome data to the multiome data. With this you can pretend everything is multiome and the code should work fine. We have never tested this however, so be careful with the results.

Hope this helps.

Best,

Seppe
